cmake/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ option(onnxruntime_USE_PREINSTALLED_EIGEN "Use pre-installed EIGEN. Need to prov
 option(onnxruntime_BUILD_BENCHMARKS "Build ONNXRuntime micro-benchmarks" OFF)
 option(onnxruntime_USE_LLVM "Build TVM with LLVM" OFF)
 
+cmake_dependent_option(onnxruntime_USE_CUTLASS "Build with Cutlass support" ON "NOT WIN32; onnxruntime_USE_CUDA" OFF)
```

Comment:
"NOT WIN32; onnxruntime_USE_CUDA" => "onnxruntime_USE_CUDA"
It is because memory efficient attention is used in windows, so we need cutlass in windows build.